stage;type;error;datetime
read_api;MissingSchema;Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?;2024-04-25 18:14:27.388459
preparation;ValueError;Invalid file path or buffer object type: <class 'NoneType'>;2024-04-25 18:14:27.392979
read_api;MissingSchema;Invalid URL 'None': No scheme supplied. Perhaps you meant https://None?;2024-04-25 18:24:37.536019
preparation;ValueError;Invalid file path or buffer object type: <class 'NoneType'>;2024-04-25 18:24:37.544549
read_api;OSError;Cannot save file into a non-existent directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/data/raw';2024-04-25 18:27:39.948319
preparation;ValueError;Invalid file path or buffer object type: <class 'NoneType'>;2024-04-25 18:27:39.952491
read_api;OSError;Cannot save file into a non-existent directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/data/raw';2024-04-25 18:30:04.401709
preparation;ValueError;Invalid file path or buffer object type: <class 'NoneType'>;2024-04-25 18:30:08.001782
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:30:39.449522
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:31:19.632992
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:32:03.719887
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:33:02.285993
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:45:46.341455
preparation;FileNotFoundError;[Errno 2] No such file or directory: '/mnt/wsl/Projects/Mba/DataOpsDevelopDeliverAnalytics/de04_dataops/python/scripts/scripts/metadado.xlsx';2024-04-25 18:46:18.402501
preparation;DatabaseError;2005 (HY000): Unknown MySQL server host 'mysql' (-2);2024-04-25 18:46:52.926557
preparation;DatabaseError;2005 (HY000): Unknown MySQL server host 'mysql' (-2);2024-04-25 18:47:02.962721
preparation;DatabaseError;2005 (HY000): Unknown MySQL server host 'mysql' (-2);2024-04-25 19:14:42.412865
preparation;DatabaseError;2005 (HY000): Unknown MySQL server host 'mysql' (-2);2024-04-25 19:16:01.618687
preparation;DatabaseError;"(mysql.connector.errors.DatabaseError) 2005 (HY000): Unknown MySQL server host 'mysql' (-2)
(Background on this error at: https://sqlalche.me/e/20/4xp6)";2024-04-25 19:16:21.946997
